bahmni_sale/models/res_config_settings.py

```diff
@@ -3,7 +3,8 @@
 class ResConfigSettings(models.TransientModel):
     _inherit = "res.config.settings"
    
-    sale_price_markup = fields.Boolean(string="Sale Price Markup Rule", store=True)
+    is_delivery_automated = fields.Boolean(string="Enable auto delivery on sale order confirm action", config_parameter="bahmni_sale.is_delivery_automated")
+    sale_price_markup = fields.Boolean(string="Sale Price Markup Rule", )
```

Comment:
This property does not have `config_parameter` option, whereas the above property has. Please make it consistent @karthikeyansp91 @hari1119